py-polars/pyproject.toml

```diff
@@ -40,10 +40,12 @@ Changelog = "https://github.com/pola-rs/polars/releases"
 [project.optional-dependencies]
 # NOTE: keep this list in sync with show_versions() and requirements-dev.txt
 adbc = ["adbc_driver_manager", "adbc_driver_sqlite"]
-async = ["nest_asyncio"]
+async = ["gevent", "nest_asyncio"]
```

Comment:
I think we should probably get rid of `async`. Those two functionalities (collect async, read from async database) aren't really related I think. Not sure here.